packages/react/src/contexts/FlowHandlerContext.ts

```diff
@@ -16,6 +17,7 @@ export interface FlowHandlerContextProps {
   currentUserState: UserState;
   currentVerificationMethod: VerificationMethods | undefined;
   initialized: boolean;
+  projectConfig: ProjectConfig | undefined;
```

Comment:
I would keep projectConfig inside FlowHandlerProvider and I would not expose it directly to the rest of the react code => Right now the FlowHandler is a layer that "translates" the projectConfig that comes from our backend to settings that can be used throughout the frontend code. 

Instead of projectConfig, I would add two new keys here: "allowSignUp" and "userFullNameRequired" and set them from FlowHandlerProvider (not a big change now but that allows us to stick to our pattern).